internal/pkg/cli/job_deploy.go

```diff
@@ -364,6 +364,9 @@ func (o *deployJobOpts) manifest() (interface{}, error) {
 	if err != nil {
 		return nil, fmt.Errorf("apply environment %s override: %s", o.envName, err)
 	}
+	if err := mft.Validate(); err != nil {
+		return nil, fmt.Errorf("validate manifest for environment %s: %s", o.envName, err)
```

Comment:
nit: if we ever ever add environment manifest then this could be confusing? Samesies elsewhere.
```suggestion
		return nil, fmt.Errorf("validate manifest with environment %s overrides: %s", o.envName, err)
```

Reply:
But they don't necessarily have env override 🤔 i'll use `against` to eliminate misunderstanding